Makefile

```diff
@@ -0,0 +1,13 @@
+build-fips:
+	GOEXPERIMENT=boringcrypto go build -tags=boringcrypto
+
+clean:
+	rm -f tyk-pump
+
+run-fips: build-fips
+	./$(BINARY_NAME)
```

Comment:
which value does it takes `BINARY_NAME` and where does this comes from? I guess we need to initialize it?

Reply:
@sredxny it was an old variable, I forgot to remove it! thanks :)